Library/Homebrew/cmd/audit.rb

```diff
@@ -226,13 +226,9 @@ def audit_class
 
     if Object.const_defined?("GithubGistFormula") && formula.class < GithubGistFormula
       problem "GithubGistFormula is deprecated, use Formula instead"
-    end
-
-    if Object.const_defined?("ScriptFileFormula") && formula.class < ScriptFileFormula
+    elsif Object.const_defined?("ScriptFileFormula") && formula.class < ScriptFileFormula
       problem "ScriptFileFormula is deprecated, use Formula instead"
-    end
-
-    if Object.const_defined?("AmazonWebServicesFormula") && formula.class < AmazonWebServicesFormula
+    elsif Object.const_defined?("AmazonWebServicesFormula") && formula.class < AmazonWebServicesFormula
       problem "AmazonWebServicesFormula is deprecated, use Formula instead"
     end
```

Comment:
Now that you're touching this piece of code: Is there any reason why those class names can't go into an array and the check modified to work with this array? It doesn't feel particularly elegant to me to repeat every class name three times and to have three copies of basically the same check. (Just an idea, feel free to ignore me.)